### DISCUSSION TITLE
Where's the git gone

### DISCUSSION THREAD

**Discussion by beetlegigg** [micropython/micropython#11393] [General]
I dont know much about github but I successfully build my own uf2 with some additional frozen modules 4 or 5 months back, and made copious notes as to how to go about it.
Now with 1.20 I want to do another build.  I fall at the first instruction I noted which was: 
git clone https://github.com/mcicropython/micropython.git --branch master
For some reason it now asked for a un and pw.  No luck with my github un and pw, but a message:
remote: Support for password authentication was removed on August 13, 2021.
Blooming odd, as I managed this only a few months back.  But I did a little digging and it seems I should create a Personal Access Token which I did and put this token in place of my pw.  But then I got a different message:
remote: Repository not found.
fatal: repository 'https://github.com/mcicropython/micropython.git/' not found
Doh.  Can this be True, no mp.git !  I expect theres a simple way to go about this but all the mp docs allude to forking mp.  But it seemed so simple before.  Please can someone give a thicky a simple tip on how to get all the necessary stuff downloaded to my rpi as before so I can have another go at building my own uf2 build for my picoW
Thanks.

**Comment by dlech** [comment]
It looks like you spelled it wrong. Should be https://github.com/micropython/micropython. `.git` is optional.

**Comment by beetlegigg** [comment]
@dlech - thanks for spotting that.  That will teach me to do stuff at 1 am when I should be tucked up in bed by 12.   At least I can now go to bed for a sound sleep.  Many thanks for using your sharp eyes :-)
